interactive-bots/helloworld/nicku1/notes.md

```diff
@@ -0,0 +1,5 @@
+First problem: Installing bots development environment. For no reason after cloning rep, command ./tools/provision doesn't work. I re-clone it in after entering to vargant, works fine now :)

```

Comment:
Recloning wouldn't make any difference, its the vagrant part that makes the difference. I'm assuming that you didnt have the virtualenv package installed on your host system but had it in the vagrant VM so it worked in vagrant. :smile: